app/android/build.gradle

```diff
@@ -34,5 +34,7 @@ allprojects {
 
         google()
         maven { url 'https://www.jitpack.io' }
+
+        jcenter()
```

Comment:
Might be worth adding a comment here so folks understand why this line is in here - although `jcenter` is deprecated, currently this is the only way to get the video component working properly on Android - this will need to be updated in the future